lib/tasks/foreman_templates_tasks.rake

```diff
@@ -0,0 +1,75 @@
+# Tasks
+namespace :templates do
+  desc 'Synchronize templates from a git repo'
+  task :sync => :environment do
+    # Available options:
+    #* verbose   => Print extra information during the run [false]
+    #* repo      => Sync templates from a different Git repo [https://github.com/theforeman/community-templates]
+    #* branch    => Branch in Git repo [default branch]
+    #* prefix    => The string all imported templates should begin with [Community ]
+    #* dirname   => The directory within the git tree containing the templates [/]
+    #* filter    => Import names matching this regex (case-insensitive; snippets are not filtered)
+    #* associate => Associate to OS, always, new or never  [new]
+
+    ForemanTemplates::TemplateImporter.new({
+      verbose:   ENV['verbose'],
+      repo:      ENV['repo'],
+      branch:    ENV['branch'],
+      prefix:    ENV['prefix'],
+      dirname:   ENV['dirname'],
+      filter:    ENV['filter'],
+      associate: ENV['associate'],
+    }).import!
+  end
+
+    ForemanTemplates::TemplateImporter.new({
```

Comment:
You've not started a new task here (:task purge ...) so when you put `end` after the block, it causes a syntax error